packages/core/webpack/config/loaders.js

```diff
@@ -77,8 +82,12 @@ module.exports = props => {
     );
   }
   else {
+    if (!props.server) {
```

Comment:
@frabarz should this be `!props.server && !production`, or is it okay for this plugin to be part of the prod build?

Reply:
uh, look again at the `if` preceding this `else` block

Reply:
🤦🏻 my bad...